tests/installation/partitioning_raid.pm

```diff
@@ -148,45 +148,71 @@ sub run() {
 
     # user defined
     send_key $cmd{custompart};
+    assert_screen 'custompart_option-selected';
     send_key $cmd{next};
     assert_screen 'custompart';
 
     send_key "tab";
-    send_key "down";    # select disks
+    assert_screen 'custompart_systemview-selected';
+    send_key "down";
+    assert_screen 'partitioning_raid-hard_disks-selected';
+
     if (get_var("OFW")) {    ## no RAID /boot partition for ppc
+        wait_idle 3;
```

Comment:
The wait idle is to get the screenshot to be able to do assert_screen. If I simply save the screenshot without waiting, I get an early wrong screenshot that I cannot use.

Notice that I only have these lines on OFW (ppc) and UEFI (aarch64), which are the two that I haven't covered yet.